test/test_torch.py

```diff
@@ -3230,6 +3230,13 @@ def test_Size(self):
         self.assertIsInstance(x[:-1], torch.Size)
         self.assertIsInstance(x + x, torch.Size)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_big_transpose(self):
```

Comment:
Can you add a comment saying that it's a unit test for this specific function? We're going to forget why do we even need this